Validation/parse_simulator.py

```diff
@@ -13,8 +13,7 @@
 PERIPHERALS: List[str] = [
     "ADC",
     "CAN",
-    "Contactor",
-    "Fan",
+    "PWM",
```

Comment:
Why is this changed?

Reply:
this is because the contactor and fan are no longer printing anything in the simulator when they are initialized, so this change was needed for the simulator check to pass